docs/howto/initialize-database.rst

```diff
@@ -93,3 +95,7 @@ Initialize the database
 
     # initialize the database
     airflow db init
+
+.. spelling::
+
+     hba
```

Comment:
We use https://github.com/apache/airflow/blob/master/docs/spelling_wordlist.txt to maintain list of exception